examples/testmonitor/results/Results.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor
+{
+    /// <summary>
+    /// Example for the SystemLink Test Monitor API that creates a test result
+    /// and associated steps simulating a current and voltage sweep.
+    /// </summary>
+    class Results
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            /*
+             * Create the TestDataManager for communicating with the server.
+             */
+            var testDataManager = new TestDataManager(configuration);
+
+            // Intialize the random number generator
+            var random = new Random();
+
+            // Set test limits
+            var lowLimit = 0;
+            var highLimit = 70;
+
+            // Initialize a ResultData object
+            var resultData = new ResultData()
+            {
+                Operator = "John Smith",
+                ProgramName = "Power Test",
+                Status = new Status(StatusType.Running),
+                SerialNumber = Guid.NewGuid().ToString(),
+                PartNumber = "NI-ABC-123-PWR"
+            };
+            // Create the test result on the SystemLink server
+            var testResult = testDataManager.CreateResult(resultData);
+            testResult.AutoUpdateTotalTime = true;
+
+            /*
+            * Simulate a sweep across a range of electrical current and voltage.
+            * For each value, calculate the electrical power (P=IV).
+            */
+            for (var current = 0; current < 10; current++)
+            {
+                // Generate a parent step to represent a sweep of voltages at a given current
+                var currentStepData = GenerateStepData($"Voltage Sweep", "SequenceCall", null, null, null);
+                // Create the step on the SystemLink server
+                var currentStep = testResult.CreateStep(currentStepData);
+
+                for (var voltage = 0; voltage < 10; voltage++)
+                {
+                    // Simulate obtaining a power measurement
+                    var (power, inputs, outputs) = MeasurePower(current, voltage);
+
+                    // Testing the power measurement
+                    var status = (power < lowLimit || power > highLimit) ? new Status(StatusType.Failed) : new Status(StatusType.Passed);
+                    var test_parameters = BuildPowerMeasurementParams(power, lowLimit, highLimit, status);
```

Comment:
```suggestion
                    var testParameters = BuildPowerMeasurementParams(power, lowLimit, highLimit, status);
```
I do this frequently - too much Python :)